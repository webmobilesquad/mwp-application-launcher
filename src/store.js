import createStore from 'unistore';

export const store = createStore({
  applications: {
    data: null,
    loading: false,
  },
});
