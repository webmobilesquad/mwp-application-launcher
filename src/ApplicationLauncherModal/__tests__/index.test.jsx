import render from 'preact-render-spy';
import { Provider } from 'unistore/preact';
import { ApplicationLauncherModal } from '../index';
import { store } from '../../store';

describe('<ApplicationLauncherModal />', () => {
  test('<ApplicationLauncherModal /> matches snapshot', () => {
    expect(
      render(
        <Provider store={store}>
          <ApplicationLauncherModal />
        </Provider>,
      ),
    ).toMatchSnapshot();
  });
});
