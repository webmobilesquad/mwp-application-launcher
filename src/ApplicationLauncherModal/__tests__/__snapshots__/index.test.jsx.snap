// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ApplicationLauncherModal /> <ApplicationLauncherModal /> matches snapshot 1`] = `
preact-render-spy (1 nodes)
-------
<aside class="ApplicationLauncherModal">
  <div class="ApplicationLauncherModal_Header">
    <button
      type="button"
      class="CloseButton"
    >
      <svg
        xmlns="http://www.w3.org/2000/svg"
        width="48"
        height="48"
        viewBox="0 0 24 24"
      >
        <path d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"></path>
        <path
          d="M0 0h24v24H0z"
          fill="none"
        >
        </path>
      </svg>
    </button>
  </div>
  <i
    class="ApplicationLauncherModal_Content"
    grid={true}
  >
  </i>
</aside>

`;
